NEWS

```diff
@@ -1,16 +1,274 @@
+  (#1108)
+
+* Added tabix --cache option to set a BGZF block cache size.  Most beneficial
+  when the -R option is used and the same blocks need to be re-read multiple
+  times.  (#1053)
+
+* Improved error checking in tabix and added a --verbosity option so
+  it is possible to change the amount of logging when it runs. (#1040)
+
+* A note about the maximum chromosome length usable with TBI indexes has been
+  added to the tabix manual page.  Thanks to John Marshall.  (#1070)
+
+* New method vcf_open_mode() changes the opening mode of a variant file
+  based on its file extension. Similar to sam_open_mode(). (#1096)
+
+* The VCF parser has been made faster and easier to maintain. (#1057)
+
+* bcf_record_check() has been made faster, giving a 15% speed increase when
+  reading an uncompressed BCF file. (#1130)
 
-* New method vcf_open_mode() changes the opening mode of a variant call file,
-  based on its file extension. Similar to sam_open_mode().
+* The VCF parser now recognises the "<NON_REF>" symbolic allele produced
+  by GATK. (#1045)
+
+* Support has been added for simultaneous reading of unindexed VCF/BCF files
+  when using the synced_bcf_reader interface.  Input files must have the
+  chromosomes in the same order as each other and be consistent with the order
+  of sequences in the header. (#1089)
+
+* The VCF and BCF readers will now attempt to fix up invalid INFO/END tags
+  where the stored END value is less than POS, resulting in an apparently
+  negative record length.  Such files have been generated by programs which
+  used END incorrectly, and by broken lift-over processes that failed to
+  update any END tags present. (#1021; fixed bcftools/#1154)
+
+* The htsFile interface can now detect the crypt4gh encrypted format (see
+  https://samtools.github.io/hts-specs/crypt4gh.pdf).  If HTSlib is
+  built with external plug-in support, and the hfile_crypt4gh plug-in is
+  present, the file will be passed to it for decryption.  The plug-in
+  can be obtained from https://github.com/samtools/htslib-crypt4gh. (#1046)
 
 * hts_srand48() now seeds the same POSIX-standard sequences of pseudo-random
   numbers regardless of platform, including on OpenBSD where plain srand48()
   produces a different cryptographically-strong non-deterministic sequence.
+  Thanks to John Marshall.  (#1002)
+
+* Iterators now work with 64 bit positions. (#1018)
+
+* Improved the speed of range queries when using BAI indexes by
+  making better use of the linear index data included in the file.
+  The best improvement is on low-coverage data. (#1031)
+
+* Alignments which consume no reference bases are now considered to have
+  length 1.  This would make such alignments cover 1 reference position in the
+  same manner as alignments that are unmapped or have no CIGAR strings.  These
+  alignments can now be returned by iterator based queries.  Thanks to John
+  Marshall. (#1063; fixed samtools/#1240, see also hts-specs#521).
```

Comment:
There is a mix of styles when referring to issues from other repos. GitHub uses `<organization>/<repo>#<no>`. If all the repos are under `samtools` organization, we should go with `<repo>#<no>`.

Reply:
Opps.  I'd noticed this but forgot to change them.  Now done.

Reply:
Previously in NEWS we've written them as the full `samtools/samtools#1240` style and hence they automatically become links in the Releases text.

I don't know if the abbreviated `repo#NN` style will be automatically linkified (you sometimes see that as a link on GH web pages, but it's usually just being displayed abbreviated from its original `org/repo#NN` when it's seen on a page from the same org).